docs/send-data/hosted-collectors/http-source/zoom.md

```diff
@@ -0,0 +1,85 @@
+---
+id: zoom
+title: Zoom Source
+description: An HTTP Zoom Source is an endpoint for receiving traces.
```

Comment:
I thought it was "Zoom HTTP Source"? What is it called in the UI?

Reply:
Yes, it is a HTTP source for Zoom app. But in UI it is just called as Zoom.